ui/raidboss/data/07-dt/ultimate/futures_rewritten.ts

```diff
@@ -1,15 +1,203 @@
+import Outputs from '../../../../../resources/outputs';
+import { Responses } from '../../../../../resources/responses';
 import ZoneId from '../../../../../resources/zone_id';
 import { RaidbossData } from '../../../../../types/data';
 import { TriggerSet } from '../../../../../types/trigger';
 
-export type Data = RaidbossData;
+export interface Data extends RaidbossData {
+  p1FallOfFaithTethers: ('fire' | 'lightning')[];
+}
 
 const triggerSet: TriggerSet<Data> = {
   id: 'FuturesRewrittenUltimate',
   zoneId: ZoneId.FuturesRewrittenUltimate,
   timelineFile: 'futures_rewritten.txt',
+  initData: () => {
+    return {
+      p1FallOfFaithTethers: [],
+    };
+  },
   timelineTriggers: [],
-  triggers: [],
+  triggers: [
+    {
+      id: 'FRU P1 Cyclonic Break Fire',
+      type: 'StartsUsing',
+      netRegex: {
+        id: ['9CD0', '9D89'],
+        source: ['Fatebreaker', 'Fatebreaker\'s Image'],
+        capture: false,
+      },
+      durationSeconds: 8,
+      alertText: (_data, _matches, output) => output.clockPairs!(),
+      outputStrings: {
+        clockPairs: {
+          en: 'Clock spots => Pairs',
+        },
+      },
+    },
+    {
+      id: 'FRU P1 Cyclonic Break Lightning',
+      type: 'StartsUsing',
+      netRegex: {
+        id: ['9CD4', '9D8A'],
+        source: ['Fatebreaker', 'Fatebreaker\'s Image'],
+        capture: false,
+      },
+      durationSeconds: 8,
+      alertText: (_data, _matches, output) => output.clockSpread!(),
+      outputStrings: {
+        clockSpread: {
+          en: 'Clock spots => Spread',
+        },
+      },
+    },
+    {
+      id: 'FRU P1 Powder Mark Trail',
+      type: 'StartsUsing',
+      netRegex: { id: '9CE8', source: 'Fatebreaker', capture: true },
+      response: Responses.tankBusterSwap(),
+    },
+    {
+      id: 'FRU P1 Utopian Sky',
+      type: 'Ability',
+      netRegex: {
+        id: ['9CDA', '9CDB'],
+        source: ['Fatebreaker', 'Fatebreaker\'s Image'],
+        capture: true,
+      },
+      delaySeconds: 6,
+      durationSeconds: 10,
+      alertText: (_data, matches, output) => {
+        if (matches.id === '9CDA')
+          return output.utopianFire!();
+        if (matches.id === '9CDB')
+          return output.utopianThunder!();
+        return output.unknownUtopian!();
+      },
+      outputStrings: {
+        utopianFire: Outputs.stacks,
+        utopianThunder: Outputs.spread,
+        unknownUtopian: Outputs.unknown,
+      },
+    },
+    {
+      id: 'FRU P1 Burnt Strike Fire',
+      type: 'StartsUsing',
+      netRegex: { source: 'Fatebreaker', id: '9CC1', capture: false },
+      durationSeconds: 8,
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Line Cleave -> Knockback',
+          de: 'Linien AoE -> Rückstoß',
+          fr: 'AoE en ligne -> Poussée',
+          ja: '直線範囲 -> ノックバック',
+          cn: '直线 -> 击退',
+          ko: '직선 장판 -> 넉백',
```

Comment:
```suggestion
          en: 'Line Cleave => Knockback',
          de: 'Linien AoE => Rückstoß',
          fr: 'AoE en ligne => Poussée',
          ja: '直線範囲 => ノックバック',
          cn: '直线 => 击退',
          ko: '직선 장판 => 넉백',
```
We should keep these consistent.